styles.css

```
@@ -117,6 +117,30 @@ body {
   margin-top: 10px;
 }
 
+.manga-item-container {
+  display: inline-block;
+  position: relative;
+}
+
+.kebab-menu {
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.5);
+  border-radius: 50%;
+  cursor: pointer;
+  display: flex;
+  height: 24px;
+  justify-content: center;
+  position: absolute;
+  right: 8px;
+  top: 8px;
+  width: 24px;
+}
+
+.kebab-menu span {
```

Rule for the class name is more consistent and makes it easier to identify
```suggestion
.kebab-menu > .material-symbols-outlined {
```